src/test/java/uk/gov/companieshouse/itemhandler/mapper/OrderDataToCertificateOrderConfirmationMapperTest.java

```diff
@@ -10,15 +10,7 @@
 import org.springframework.test.context.junit.jupiter.SpringExtension;
 import org.springframework.test.context.junit.jupiter.SpringJUnitConfig;
 import uk.gov.companieshouse.itemhandler.email.CertificateOrderConfirmation;
-import uk.gov.companieshouse.itemhandler.model.ActionedBy;
-import uk.gov.companieshouse.itemhandler.model.CertificateItemOptions;
-import uk.gov.companieshouse.itemhandler.model.CertificateType;
-import uk.gov.companieshouse.itemhandler.model.DeliveryDetails;
-import uk.gov.companieshouse.itemhandler.model.DeliveryTimescale;
-import uk.gov.companieshouse.itemhandler.model.DirectorOrSecretaryDetails;
-import uk.gov.companieshouse.itemhandler.model.Item;
-import uk.gov.companieshouse.itemhandler.model.OrderData;
-import uk.gov.companieshouse.itemhandler.model.RegisteredOfficeAddressDetails;
+import uk.gov.companieshouse.itemhandler.model.*;
```

Comment:
We should not be importing using the wildcard, items should be imported individually.